### PR TITLE
Add Sphinx Docs

### PR DESCRIPTION
**Pull request by sh4nks**
I just started working on this. Any thoughts so far? Would you be cool if I change the docstrings inside the methods so that the params are also transformed into some fancy html?

This is still a WIP.

### CODE REVIEW

**Comment by bkabrda** [comment]
This looks awesome so far!
I'm all for changing the docstrings, but since your yesterday comment made me work on this: https://github.com/bkabrda/flask-whooshee/pull/28, I'd like to ask you to wait, so that neither of us have to rebase their PRs. Feel free to review/comment that PR if you have some time, I'd like to hear what you think about it. If nothing breaks, I assume I'll merge it on Monday, latest.

**Comment by sh4nks** [comment]
Ok I'll wait with this stuff and just try to add some docs for now :)

**Comment by bkabrda** [comment]
The PR was merged, so feel free to go on with the docstrings!

File: setup.py [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+Adds Whoosh support to Flask-SQLAlchemy.
 
+Links
+`````
+
+* `Documentation <https://pythonhosted.org/Flask-Whooshee/>`_
```
Comment (bkabrda) [inline setup.py]: I have to say I'd prefer readthedocs.io - pythonhosted doesn't allow automatic documentation builds on every push AFAICT. I registered the url for flask-whooshee right now, so let's use it: http://flask-whooshee.readthedocs.io/en/latest/

**Comment by sh4nks** [comment]
Mhh looks like I fucked something up. Should I reopen the PR?

Besides that, I would be finished :) Any suggestions/improvements?

**Comment by bkabrda** [comment]
The content is perfect! As for the commits problem, it seems that something has gone wrong during the rebase, but I'm not sure. Opening a new PR would likely be the best option, since Github won't allow me to merge this one anyway (complains about `flask_whooshee.py` as a conflicting file.

**Pull request closed without merging**